### PR TITLE
Just some cleanup

### PR DESCRIPTION
**Pull request by dented42**
I converted modules to the 'new' `#lang` syntax and cleaned up some require/provide specs.

### CODE REVIEW

File: core/dv.rkt [hunk 1]
```diff
@@ -44,17 +57,4 @@
...
- [dv:length (dv? . -> . exact-nonnegative-integer?)]
- [dv:remove-last (non-empty-dv? . -> . void)]
- [dv:ref (->d ([dv dv?] [pos exact-nonnegative-integer?]) () 
-              #:pre-cond (pos . < . (dv:length dv))
-              [r any/c])]
- [dv:set! (->d ([dv dv?] [pos exact-nonnegative-integer?] [val any/c]) () 
-               #:pre-cond (pos . < . (dv:length dv))
-               [r void])]
- [dv:append (dv? any/c . -> . void)])
+  ((dv:length dv) . > . 0))
```
Comment (ghcooper) [inline core/dv.rkt]: Any reason not to have an end-of-file newline?

File: core/frp.rkt [hunk 2]
```diff
@@ -675,56 +727,4 @@
...
- [super-lift ((any/c . -> . any/c) any/c . -> . any/c)] ; XXX Ugly contract
- [behavior? (any/c . -> . boolean?)]
- [value-now (any/c . -> . any/c)] ; XXX Should this return (not/c signal?) and why not take signal?
- [value-now/sync (() () #:rest (listof any/c) . ->* . any)] ; XXX See above + not matching number of values returned with number of signals
- [value-now/no-copy (any/c . -> . any/c)] ; XXX Should this return (not/c signal?) and why not take signal?
- [safe-signal-depth (any/c . -> . exact-nonnegative-integer?)] ; XXX Ugly contract
- [schedule-alarm (number? signal? . -> . void)]
- [set-cell! (signal? any/c . -> . void)] ; XXX What is any/c?
- [exceptions signal?])
+  any/c)
```
Comment (ghcooper) [inline core/frp.rkt]: Add newline?

Comment (bennn) [reply core/frp.rkt]: The official style guide says "End files with a newline": http://docs.racket-lang.org/style/Textual_Matters.html#%28part._.End_of_.File%29

Comment (dented42) [reply core/frp.rkt]: Oops... That's just my own personal dislike of trailing newlines, I usually try to audit contributions to add back any that I instinctually removed but I forgot this time. :p

Sorry about that.

**Comment by dented42** [comment]
This should be more to your liking. :)

**Pull request merged**